### PR TITLE
scorep/30+cuda8 for cosmo

### PR DESCRIPTION
**Pull request by jgphpc**

### CODE REVIEW

**Comment by jgphpc** [comment]
https://jenkins.cscs.ch/job/ProductionEBtestingPR-gpu/245/label=daint/console 👍 
`export SCOREP_CUDA_ENABLE=yes` 🆗

**Comment by jgphpc** [comment]
tested.

File: easybuild/easyconfigs/s/Score-P/Score-P-3.0-CrayGNU-2016.11-cuda-8.0.54.eb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+sanity_check_paths = {
+    'files': ["bin/scorep", "include/scorep/SCOREP_User.h",
+             ("lib/backend/libscorep_adapter_mpi_event.a",
+              "lib/libscorep_adapter_mpi_event.%s" % SHLIB_EXT) ],
+    'dirs': [],
+}
+
+# Ensure that local metric documentation is found by Cube GUI
+modextrapaths = { 'CUBE_DOCPATH': 'share/doc/scorep/profile' }
+modextravars = { 'CRAYPE_LINK_TYPE' : 'dynamic' }
```
Comment (gppezzi) [inline easybuild/easyconfigs/s/Score-P/Score-P-3.0-CrayGNU-2016.11-cuda-8.0.54.eb]: Dynamic builds are the default with EasyBuild, you could remove this. This variable also set if cudatoolkit is loaded.

Comment (jgphpc) [inline easybuild/easyconfigs/s/Score-P/Score-P-3.0-CrayGNU-2016.11-cuda-8.0.54.eb]: https://github.com/eth-cscs/production/pull/105/commits/9a90bf3a6ff60fd6465128f30a7ed485fe78473e

**Pull request merged**